decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDecider.java

```diff
@@ -311,14 +343,14 @@ public void setConfig(Map<String, String> config) {
         return validWorkflowRuns;
     }
 
-    public List<WorkflowRun> getInvalidWorkflowRuns() {
-        return invalidWorkflowRuns;
-    }
-
     public List<WorkflowRun> getScheduledWorkflowRuns() {
         return scheduledWorkflowRuns;
     }
 
+    public List<String> getInvalidLanes() {
+        return laneErrors;
+    }
+
     public List<WorkflowRun> run() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 165 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)